cli/file/definition_test.go

```diff
@@ -131,10 +133,55 @@ func TestLoadDefinition(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:          "Should_parse_env_variables",
+			File:          "../testdata/definitions/valid_http_test_definition_with_env_variables.yml",
+			ShouldSucceed: true,
+			EnvVariables: map[string]string{
+				"POKEMON_APP_API_KEY": "my secret key",
+			},
+			ExpectedDefinition: definition.Test{
+				Name:        "POST import pokemon",
+				Description: "Import a pokemon using its ID",
+				Trigger: definition.TestTrigger{
+					Type: "http",
+					HTTPRequest: definition.HttpRequest{
+						URL:    "http://pokemon-demo.tracetest.io/pokemon/import",
+						Method: "POST",
+						Headers: []definition.HTTPHeader{
+							{Key: "Content-Type", Value: "application/json"},
+						},
+						Body: definition.HTTPBody{
+							Type: "raw",
+							Raw:  `{ "id": 52 }`,
+						},
+						Authentication: definition.HTTPAuthentication{
+							Type: "apiKey",
+							ApiKey: definition.HTTPAPIKeyAuth{
+								Key:   "X-Key",
+								Value: "my secret key",
+								In:    "header",
+							},
+						},
+					},
+				},
+			},
+		},
 	}
 
 	for _, testCase := range testCases {
+		envVariables := testCase.EnvVariables
 		t.Run(testCase.Name, func(t *testing.T) {
+			for key, value := range envVariables {
+				os.Setenv(key, value)
```

Comment:
This will prevent this test to work parallel. Each parallel case might override the other at the same time, since `os.Setenv` is "global", so either refactor the code to have an `EnvGetter` injected, or totally ignore the parallel tests, since this is a cli, and parallelization shouldn't really be a problem

Reply:
Decided to ignore the parallel tests.